accessories/light.js

```diff
@@ -37,16 +37,47 @@ HomeAssistantLight.prototype = {
     }),
     is_supported: function(feature) {
         // If the supported_features attribute doesn't exist, assume supported
-        return this.data.attributes.supported_features === undefined ||
-      ((this.data.attributes.supported_features & feature) > 0);
+        if (this.data.attributes.supported_features === undefined) {
+            return true;
+        }
+
+        if ((this.data.attributes.supported_features & feature) > 0) {
+            //workaround because homeassistant includes RGB_COLOR feature in zwave lights even if they haven't RGB
+            //https://github.com/home-assistant/home-assistant/issues/5333
+            if (feature == this.features.RGB_COLOR && this.data.attributes.rgb_color === undefined) {
```

Comment:
This caused an issue with one of my (non-Hue) lights (a flux_led platform light). The light wasn't detected by Homebridge because it was off when I started Homebridge. After stopping Homebridge, turning the light on, and starting Homebridge again, it showed up. Is there any way around this?